.gitignore

```diff
@@ -196,3 +196,6 @@ $RECYCLE.BIN/
 
 # ignore left over error files
 errorlogfile.json
+
+# test input
+large-space.json
```

Comment:
Should we really have this permanently in the gitignore? :)

Reply:
anyway - deleted it!